test/unit/specs/composables/use-i18n-utilities-in-component.spec.js

```diff
@@ -0,0 +1,42 @@
+import VueI18n from 'vue-i18n'
+import { render, screen } from '@testing-library/vue'
+import { AUDIO, IMAGE } from '~/constants/media'
+import VContentLink from '~/components/VContentLink/VContentLink.vue'
+
+const enMessages = require('~/locales/en.json')
+const i18n = new VueI18n({
+  locale: 'en',
+  fallbackLocale: 'en',
+  messages: { en: enMessages },
+})
+
+describe('i18nResultsCount in a component', () => {
+  let options = {}
+  let count
+  beforeEach(() => {
+    options = {
+      props: { mediaType: 'image', resultsCount: count },
+      mocks: { $nuxt: { context: { i18n } } },
+    }
+  })
+
+  it.each`
+    resultCount | mediaType | expectedText
+    ${0}        | ${IMAGE}  | ${'No image results'}
+    ${0}        | ${AUDIO}  | ${'No audio results'}
+    ${10}       | ${IMAGE}  | ${'10 image results'}
+    ${10}       | ${AUDIO}  | ${'10 audio results'}
+    ${10000}    | ${IMAGE}  | ${'Over 10,000 image results'}
+    ${10000}    | ${AUDIO}  | ${'Over 10,000 audio results'}
+  `(
+    'Should show correct result for $resultCount of type $mediaType',
```

Comment:
Thanks for adding these tests! How do they differ from the other tests you added that test the composable in isolation? They seem to test the same things, just in different contexts. With this test in particular it would break if `VContentLink` stopped displaying the count for any reason, which seems weird.

If we need to test it in a component, then a test specific component created in this file might be a better option to avoid that kind of thing.

Reply:
I guess after the comment on the original issue that we don't actually need the 'image/audio results' string in the `VContentLink`, I will simply remove it. I used this test to use the nuxt `app.i18n.tc` that is not available outside of a component, and using a component defined especially for test for testing Nuxt-specific things can be tricky.

Reply:
Ahh gotcha, that makes sense!